app/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+    defaultConfig {
+        applicationId "org.fossasia.fossasiaorgaandroidapp"
```

Comment:
Let the package name be 
`org.fossasia.openevent.app`. 

Please update all the classes accordingly too.